botocore/serialize.py

```diff
@@ -486,6 +495,15 @@ def _partition_parameters(self, partitioned, param_name,
             elif isinstance(param_value, bool):
                 partitioned['query_string_kwargs'][
                     key_name] = str(param_value).lower()
+            elif member.type_name == 'timestamp':
+                timestamp_format = member.serialization.get('timestampFormat')
+                if timestamp_format is None:
+                    timestamp_format = self.QUERY_STRING_TIMESTAMP_FORMAT
+                timestamp_format = timestamp_format.lower()
```

Comment:
Don't think we need this, you made `_convert_timestamp_to_str` take care of lower-casing.